application/classes/Ushahidi/Validator/Form/Update.php

```diff
@@ -31,12 +31,27 @@ public function __construct(FormRepository $repo)
 
 	protected function getRules()
 	{
+
+			// Always check validation for name and description
+			$fullData = $this->validation_engine->getFullData();
+			$data = $this->validation_engine->getData();
+			if ($fullData['name']) {
+				$data['name'] = $fullData['name'];
+			}
+			if ($fullData['description']) {
+				$data['description'] = $fullData['description'];
+			}
+			$this->validation_engine->setData($data);
+			// End
+
 		return [
 			'name' => [
+				['not_empty'],
 				['min_length', [':value', 2]],
 				['regex', [':value', Validator::REGEX_STANDARD_TEXT]], // alpha, number, punctuation, space
 			],
 			'description' => [
+				['not_empty'],
 				['is_string'],
```

Comment:
Same as above, if the field is left empty, `min_length` does not throw the error, allowing for the description to be saved as an empty string, which is why I added `not_empty`. This can be removed if description should not be required.